### PR TITLE
Upgrade some DSC resources to latest stable version

### PR DESCRIPTION
**Pull request by stehlih**
# Pull Request

### Changed
- Upgrade some DSC resources to latest stable version
- Refactoring of SqlPermissions after upgrade of SqlServerDsc to 16.0.0
- Update Documentation

## Task list

- [X] The PR represents a single logical change. i.e. Cosmetic updates should go in different PRs.
- [X] Added an entry under the Unreleased section of in the CHANGELOG.md as per [format](https://keepachangelog.com/en/1.0.0/).
- [X] Local clean build passes without issue or fail tests (`build.ps1 -ResolveDependency`).
- [X] Resource documentation added/updated in README.md.
- [ ] Resource parameter descriptions added/updated in README.md, schema.mof
      and comment-based help.
- [ ] Comment-based help added/updated.
- [ ] Localization strings added/updated in all localization files as appropriate.
- [ ] Examples appropriately added/updated.
- [X] Unit tests added/updated. See [DSC Resource Testing Guidelines](https://github.com/PowerShell/DscResources/blob/master/TestsGuidelines.md).
- [ ] Integration tests added/updated (where possible). See [DSC Resource Testing Guidelines](https://github.com/PowerShell/DscResources/blob/master/TestsGuidelines.md).
- [ ] New/changed code adheres to [DSC Resource Style Guidelines](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md) and [Best Practices](https://github.com/PowerShell/DscResources/blob/master/BestPractices.md).

- - -
This change is [Reviewable](https://reviewable.io/reviews/dsccommunity/commontasks/190)

### CODE REVIEW

File: tests/Unit/DSCResources/DscResources.Tests.ps1 [hunk 1]
```diff
@@ -2,7 +2,8 @@ BeforeDiscovery {
     $dscResources = Get-DscResource -Module $moduleUnderTest.Name
     $here = $PSScriptRoot
 
-    $skippedDscResources = 'PowerShellRepositories', 'RemoteDesktopCollections', 'RemoteDesktopDeployment'
+    # SqlPermissions are in conflict with the Scom* resources
```
Comment (nyanhp) [inline tests/Unit/DSCResources/DscResources.Tests.ps1:5]: Why @stehlih, what was the issue? The SCOM resources do not make use of SQL Permissions

**Review by nyanhp — CHANGES_REQUESTED** [review]
Hi @stehlih , could you elaborate on your issue that led you to skip testing SqlPermissions?

File: tests/Unit/DSCResources/DscResources.Tests.ps1 (hunk 1 above)
Comment (stehlih) [reply tests/Unit/DSCResources/DscResources.Tests.ps1:5]: See https://github.com/nyanhp/cScom/issues/1.
The problem appears only if both resources are used in the same DSC configuration.

Comment (nyanhp) [reply tests/Unit/DSCResources/DscResources.Tests.ps1:5]: Right, got it.

**Review by nyanhp — APPROVED** [review]

**Pull request merged**